ndmg/stats/qa_skullstrip.py

```diff
@@ -0,0 +1,272 @@
+                atl = ndimage.rotate(origional[:, pos, :], 90)
+            else:
+                image = ndimage.rotate(b0[:, :, pos], 270)
+                atl = ndimage.rotate(origional[:, :, pos], 270)
+
+            if idx % 3 == 1:
+                ax.set_ylabel(labs[i])
+                ax.yaxis.set_ticks([0, image.shape[0] / 2, image.shape[0] - 1])
+                ax.xaxis.set_ticks([0, image.shape[1] / 2, image.shape[1] - 1])
+            if edge:
+                image = edge_map(image).data
+                image[image > 0] = max_val
+                image[image == 0] = min_val
+            # Set the axis invisible
+            plt.xticks([])
+            plt.yticks([])
+
+            # Set the frame invisible
+            ax.spines['top'].set_visible(False)
+            ax.spines['right'].set_visible(False)
+            ax.spines['bottom'].set_visible(False)
+            ax.spines['left'].set_visible(False)
+            ax.imshow(atl, interpolation="none", cmap=cmaps[0], alpha=0.9)
+            ax.imshow(
+                opaque_colorscale(
+                    cmaps[1], image, alpha=0.9, vmin=min_val, vmax=max_val
+                )
+            )
+            if idx ==3:
+                plt.plot(0, 0, "-", c="purple", label='skull')
+                plt.plot(0, 0, "-", c="green", label='brain')
+                # box = ax.get_position()
+                # ax.set_position([box.x0, box.y0, box.width, box.height*0.8])
+                plt.legend(loc='best', fontsize=15, frameon=False, bbox_to_anchor=(1.5, 1.5))
+
+    # Set title for the whole picture
+    [a, b, c] = ori_shape
+    title = 'QA For skullstrip. Brain Volume:' + str(a) + '*' + str(b) + '*' + str(c) + '\n'
+    foverlay.suptitle(title, fontsize=24)
+    foverlay.set_size_inches(12.5, 10.5, forward=True)
+    return foverlay
+
+def get_true_volume(nparray):
+    """
+    because there is some spacing out side of brain, instead of simply choosing
+    1/4  1/2  3/4 part of all space to slide,
+    this function is first detect where is brain and then get 1/4 1/2 3/4 part of brain
+    """
+    img_arr = nparray.astype(int)
+    threshold = int(1)
+    img_arr[img_arr <= threshold] = 0
+    img_arr[img_arr > threshold] = 1
+    true_volume = np.where(img_arr == 1)
+    x = get_range(true_volume, 0)
+    y = get_range(true_volume, 1)
+    z = get_range(true_volume, 2)
+    return x, y, z
+
+def get_range(array,i):
+    min_num = min(array[i])
+    max_num = max(array[i])
+    arrange = np.arange(min_num, max_num)
+    quarter = np.percentile(arrange, [25, 50, 75]).astype(int)
+    return quarter
+
+def get_min_max(data, minthr=2, maxthr=95):
+    """
+    data: regbrain data to threshold.
+    """
+    min_val = np.percentile(data, minthr)
+    max_val = np.percentile(data, maxthr)
+    return (min_val.astype(float), max_val.astype(float))
+
+def get_braindata(brain_file):
+    """Opens a brain data series for a mask, mri image, or atlas.
```

Comment:
This function already exists in ndmg.utils.gen_utils.get_braindata, delete this function and use the gen_utils function

Reply:
@luochuankai-JHU - you haven't fixed this yet, even though you told us to re-review.

Reply:
in the process of doing that, it's a bit better practice to [use `isinstance`](http://docs.quantifiedcode.com/python-anti-patterns/readability/do_not_compare_types_use_isinstance.html?highlight=isinstance) instead of `type` for what this function is doing, so replace with that (although this is a low-priority for you)

Reply:
e.g., 
`if type(brain_file) is str or type(brain_file) is str` can be replaced with
`if isinstance(brain_file, str)`

Reply:
fixed it in the latest version

Reply:
this code is already in qa_reg.py   so I decide to import this function from qa_reg